server/src/main/kotlin/com/xebia/functional/xef/server/http/routes/Requests.kt

```diff
@@ -0,0 +1,38 @@
+package com.xebia.functional.xef.server.http.routes
+
+import io.ktor.resources.*
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
```

Comment:
Are these models in use? If we are using the ones from the openai client lib we can get rid of these.